widget/embedded/src/pages/LanguagePage.tsx

```diff
@@ -12,11 +18,11 @@ enum Languages {
   SPANISH = 'spanish',
 }
 
-interface SettingItemPropsTypes {
+interface TitleContainerProps {
   title: string;
 }
 
-function LanguageItemTitle({ title }: SettingItemPropsTypes) {
+function TitleContainer({ title }: TitleContainerProps) {
```

Comment:
1. Please don't destructure the params directly, we don't usually do that.
2. `i18n.t(title)` don't add any value here because it's dynamic and can generate any localized string. You can use the `title` directly. 

Same comment should be applied for other pages.

Reply:
Done